fog/ledger/server/tests/router_connection.rs

```diff
@@ -426,7 +427,10 @@ fn fog_ledger_key_images_test(logger: Logger) {
                 watcher_db: watcher_dir,
                 admin_listen_uri: admin_listen_uri.clone(),
                 client_listen_uri: client_listen_uri.clone(),
-                client_responder_id: ResponderId::from_str(&client_listen_uri.addr()).unwrap(),
+                shard_uris: vec![],
```

Comment:
Yeah this is a bit weird. It's empty because that field isn't used except in the `bin/router.rs` binary - the FLR constructor expects the GRPC clients. I eventually did what Millie did and implemented a `new_from_config` which contains the shard URI handling logic, but it wasn't in this PR (would have been in the next one). I'll pull it forward and make this make sense.